x-pack/plugins/timelines/public/components/t_grid/toolbar/fields_browser/field_browser.tsx

```diff
@@ -24,46 +25,30 @@ import type { BrowserFields, ColumnHeaderOptions } from '../../../../../common';
 import { isEscape, isTab, stopPropagationAndPreventDefault } from '../../../../../common';
 import { CategoriesPane } from './categories_pane';
```

Comment:
On this file you want to clean up similar euiTheme tokens. I'd also suggest using `eui-yScroll` CSS helpers rather than writing manual overflows. This will clean up the coloring / sizing of the overflow scrollbars to look more like the rest of Kibana.

You can find the docs / examples for them here. https://elastic.github.io/eui/#/utilities/css-utility-classes